pkg/Library/NodeInspector/NodeInspectorNode.js

```diff
@@ -4,9 +4,9 @@
  * Use of this source code is governed by a BSD-style
  * license that can be found in the LICENSE file.
  */
-export const InspectorNode = {
+export const NodeInspectorNode = {
```

Comment:
sorry, i think i messed up during the merge after all :(

Reply:
I changed all the names 17 times, so ...